drivers/nutdrv_qx.c

```diff
@@ -1891,9 +1891,8 @@ static int	armac_command(const char *cmd, char *buf, size_t buflen)
 		memset(tmpbuf, 0, sizeof(tmpbuf));
 		tmpbuf[0] = 0xa0 + bytes_to_send;
 		memcpy(tmpbuf + 1, cmd + i, bytes_to_send);
-		ret = usb_control_msg(udev,
-			USB_ENDPOINT_OUT + USB_TYPE_CLASS + USB_RECIP_INTERFACE,
-			0x09, 0x200, 0,
+		ret = usb_interrupt_write(udev,
+			USB_ENDPOINT_OUT | 2,
```

Comment:
I was looking in code  for any interrupt write usages and found one in nutdrv_qx.c: https://github.com/networkupstools/nut/blob/master/drivers/nutdrv_qx.c#L1504-L1505 It uses same `USB_ENDPOINT_OUT | 2`. As it worked for me I did not put too much thought into it.

Not sure if it could be related but `lsusb -v` has following fragment
```
      Endpoint Descriptor:
        bLength                 7
        bDescriptorType         5
        bEndpointAddress     0x02  EP 2 OUT
        bmAttributes            3
          Transfer Type            Interrupt
          Synch Type               None
          Usage Type               Data
        wMaxPacketSize     0x0040  1x 64 bytes
        bInterval               1
```
And ` bEndpointAddress     0x02  EP 2 OUT` probably explains `USB_ENDPOINT_OUT | 2`. And I'm not use if we can trust that but `Transfer Type            Interrupt` probably also could be used to detect if we wan't to use interrupts or commands?